core/store/src/db.rs

```diff
@@ -85,13 +134,40 @@ pub enum DBCol {
     ColReceiptIdToShardId = 27,
     _ColNextBlockWithNewChunk = 28,
     _ColLastBlockWithNewChunk = 29,
```

Comment:
Could we document here that these two columns are deprecated?

Reply:
done